components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceConstants.java

```diff
@@ -82,7 +82,7 @@
     /**
      * This is the character used in lifecycle property keys
      */
-    public static final String DOT = ".";
+    public static final String LIFECYCLE_PROPERTY_SEPERATOR = ".";
```

Comment:
Changing the key of existing public constant will break usages. shall we add new constant and deprecate the exist one. rather changing the key.